src/Ext/WeaponType/Hook.EBolt.cpp

```diff
@@ -27,6 +29,19 @@ DEFINE_HOOK(0x4C2951, EBolt_DTOR, 0x5)
 	return 0;
 }
 
+DEFINE_HOOK(0x4C20BC, EBolt_DrawArcs, 0xB)
+{
+	enum { DoLoop = 0x4C20C7, Break = 0x4C2400 };
+
+	GET_STACK(EBolt*, pBolt, 0x40);
+	BoltTemp::pType = BoltTemp::boltWeaponTypeExt.get_or_default(pBolt);
+
+	GET_STACK(int, plotIndex, STACK_OFFSET(0x408, -0x3E0))
```

Comment:
```suggestion
	GET_STACK(EBolt*, pBolt, 0x40);
	GET_STACK(int, plotIndex, STACK_OFFSET(0x408, -0x3E0));
	BoltTemp::pType = BoltTemp::boltWeaponTypeExt.get_or_default(pBolt);
```

In most cases we group GET macros together.